website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -0,0 +1,118 @@
+      frequency_interval = 4
+      frequency_type     = "weekly"
+      retention_unit     = "weeks"
+      retention_value    = 3
+    }
+    policy_item {
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.3.id
+      frequency_interval = 5
+      frequency_type     = "monthly"
+      retention_unit     = "months"
+      retention_value    = 4
+    }
+  }
+}
+
+data "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
+  project_id   = mongodbatlas_cloud_provider_snapshot_backup_policy.test.project_id
+  cluster_name = mongodbatlas_cloud_provider_snapshot_backup_policy.test.cluster_name
+}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) The unique identifier of the project for the Atlas cluster.
+* `cluster_name` - (Required) The name of the Atlas cluster that contains the snapshots backup policy you want to retrieve.
+* `reference_hour_of_day` - (Optional) UTC Hour of day between 0 and 23, inclusive, representing which hour of the day that Atlas takes snapshots for backup policy items.
+* `reference_minute_of_hour` - (Optional) UTC Minutes after referenceHourOfDay that Atlas takes snapshots for backup policy items. Must be between 0 and 59, inclusive.
+* `restore_window_days` - (Optional) Number of days back in time you can restore to with point-in-time accuracy. Must be a positive, non-zero integer.
+* `update_snapshots` - (Optional) Specify true to apply the retention changes in the updated backup policy to snapshots that Atlas took previously.
+
+### Policies
+* `policies` - (Required) Contains a document for each backup policy item in the desired updated backup policy.
+* `policies.#.id` - (Required) Unique identifier of the backup policy that you want to update.
+
+#### Policy Item
+* `policies.#.policy_item` - (Required) Array of backup policy items.
+* `policies.#.policy_item.#.id` - (Required) Unique identifier of the backup policy item.
+* `policies.#.policy_item.#.frequency_interval` - (Required) Desired frequency of the new backup policy item specified by frequencyType.
+* `policies.#.policy_item.#.frequency_type` - (Required) Frequency associated with the backup policy item. One of the following values: hourly, daily, weekly or monthly.
+* `policies.#.policy_item.#.retention_unit` - (Required) Scope of the backup policy item: days, weeks, or months.
+* `policies.#.policy_item.#.retention_value` - (Required) Value to associate with retentionUnit.
+
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `cluster_id` - Unique identifier of the Atlas cluster.
+* `next_snapshot` - Timestamp in the number of seconds that have elapsed since the UNIX epoch when Atlas takes the next snapshot.
+
+## Import
+
+Cloud Provider Snapshot Backup Policy entries can be imported using project project_id, cluster_name, in the format `PROJECTID-CLUSTERNAME`, e.g.
```

Comment:
Would someone need to import if the cluster already existed or would they ever need to import since really cluster is always doing that for them?

Reply:
The user could import it to change the policies configuration it the cluster is already exist

Reply:
Ah, okay I need to make sure I get the logic here.  The policy will always exist if they have cloud provider snapshots enabled so this would be needed if they did not create the cluster in terraform and wanted to import both the cluster and the policy or if they use terraform before we added this?

Reply:
For import, the provider backup enabled must be enabled for the cluster if not the won't get any policy data. If the user wants to use the new resource so they want to enable the provider backup and make `terraform refresh` to get the info in the cluster.

Reply:
We should probably note that in the documentation then.  Perhaps in the provider_backup_enabled doc we add (For example: If setting to true for an existing cluster or imported cluster be sure to run `terraform refresh` after applying to enable modification of the Cloud Provider Snapshot Backup Policy going forward.)

Reply:
that's right let me add it